src/lib/wallet.js

```diff
@@ -29,25 +30,26 @@ if (IN_FRAME) {
 }
 
 export default async function initSdk() {
+  const { activeNetwork } = useNetworks();
   const { getAeSdk, resetNode } = useAeSdk({ store });
-
-  const { getMiddleware } = useMiddleware({ store });
+  const { getMiddleware } = useMiddleware();
 
   const [aeSdk] = await Promise.all([getAeSdk(), getMiddleware()]);
 
   if (IN_FRAME && !FramesConnection.initialized) {
     FramesConnection.init(aeSdk);
   }
 
-  store.watch(
-    (state, getters) => getters.activeNetwork,
+  watch(
+    activeNetwork,
     async (newValue, oldValue) => {
       if (aeSdkBlocked || isEqual(newValue, oldValue)) {
         return;
       }
       try {
         aeSdkBlocked = true;
         await resetNode(oldValue, newValue);
+        store.commit('initTransactions');
```

Comment:
This is a rebase mistake.

Reply:
No. This line was present in `switchNetwork` Vuex action. As the networks logic is not present in the Vuex I moved this here. Is this OK in your opinion?

Reply:
Sorry, it is no longer true only on the `develop` branch. We would need to remove that when merging with/rebasing on `develop`.